plugins/scaffolder-backend/src/scaffolder/tasks/convertZodtoJson.ts

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Think this file should be under the `/actions` directory not the `/tasks` folder that's something different.